compiler/src/dmd/e2ir.d

```diff
@@ -1301,6 +1301,16 @@ elem* toElem(Expression e, IRState *irs)
             }
             e = el_combine(ezprefix, e);
         }
+        else if (auto taa = t.isTypeAArray())
```

Comment:
I don't understand why Codecov says this isn't covered, it has to be or else the runnable test wouldn't work right?